src/ducks/Studio/url.js

```diff
@@ -1,14 +1,24 @@
 import { stringify, parse } from 'query-string'
 import { DEFAULT_SETTINGS, DEFAULT_OPTIONS } from './defaults'
-import { Events, Metrics, compatabilityMap } from '../SANCharts/data'
+import { Events, Metrics, compatabilityMap } from '../SANCharts/metrics/data'
 import { buildCompareKey } from './Compare/utils'
+import { Submetrics } from '../SANCharts/metrics/submetrics'
```

Comment:
**[D-1-1]** Keep the correct order of import statements - similar paths (like `'../SANCharts/metrics/`) paths should be placed close to each other